frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java

```diff
@@ -151,7 +151,6 @@ public void run(ProgressIndicator indicator) {
       @Override
       @UIEffect
       public void onSuccess() {
-        baseEnhancedGraphTable.queueRepositoryUpdateAndModelRefresh();
```

Comment:
`BaseEnhancedGraphTable` params are typically just called `graphTable`, rename while you're here

Reply:
Also... change `runWriteActionOnUIThread` to `blockRunWriteActionOnUIThread` above... now the entire construct doesn't seem to have much sense

Reply:
Yeah, generally backgroundable in the current form seems redundant TBH, to be fixed within #1465